packages/sanity/src/core/studio/components/navbar/free-trial/DescriptionSerializer.tsx

```diff
@@ -124,28 +190,42 @@ const components: PortableTextComponents = {
     iconAndText: (props) => (
       <Flex align="flex-start" paddingX={2} paddingTop={1} paddingBottom={2} marginTop={2} gap={2}>
         <Flex gap={2} style={{flexShrink: 0}}>
-          <Text size={1}>
-            <DynamicIcon icon={props.value.icon} />
-          </Text>
-          <Text size={1} weight="semibold">
+          <IconTextContainer size={1} accent={props.value.accent}>
+            {props.value.sanityIcon ? (
+              <Icon symbol={props.value.sanityIcon} />
+            ) : (
+              <DynamicIcon icon={props.value.icon} />
+            )}
+          </IconTextContainer>
+          <Text size={1} weight="semibold" accent={props.value.accent}>
             {props.value.title}
           </Text>
         </Flex>
 
-        <Text size={1} muted>
+        <Text size={1} muted accent={props.value.accent}>
           {props.value.text}
         </Text>
       </Flex>
     ),
   },
 }
 
+interface DescriptionSerializerProps {
+  blocks: PortableTextBlock[]
+}
+
+/**
+ * Portable text serializer for the description text for upsell elements.
```

Comment:
Since this serializer is intended to be used for upsell blocks – should the name of the component be more specific to better reflects its purpose? Maybe something like `UpsellDescriptionSerializer`

Reply:
Yeap, sounds good, I moved it to a reusable location instead of navbar and renamed it.